packages/driver/src/cy/assertions.coffee

```diff
@@ -111,6 +111,10 @@ create = (state, queue, retryFn) ->
 
     state("upcomingAssertions", cmds)
 
+    ## we're applying the default assertion in the
+    ## case where there are no upcoming assertion commands
+    isDefaultAssertionErr = cmds.length is 0
```

Comment:
is this used anywhere?

Reply:
no - but it could / should / needs to be which is why i sent it in. other assertion code could be refactored to utilize it which would clean up a bunch of other areas